examples/configs_plugin/main.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"time"
+
+	"github.com/ligato/cn-infra/config"
+	"github.com/ligato/cn-infra/core"
+	"github.com/ligato/cn-infra/flavors/local"
+	"github.com/ligato/cn-infra/logging/logroot"
+	"github.com/namsral/flag"
+)
+
+const PluginName = "example"
+
+// ExampleConfFlag used as flag name (see implementation in declareFlags())
+// It is used to load configuration of Example plugin.
+// This flag name is calculated from the name of the plugin.
+const ExampleConfFlag = PluginName + config.FlagSuffix
+
+// ExampleConf is default (flag value) - filename for the configuration.
+const ExampleConf = PluginName + ".conf"
```

Comment:
Name the variable as ExampleConfDefault to match the naming scheme used by DirFlag/DirDefault/DirUsage (and don't forget to update the comment).